styles/style.css

```diff
@@ -0,0 +1,43 @@
+html {
+    font-size: 10px; 
+    font-family: 'Didact Gothic', Tahoma, sans-serif;
+}
+
+h1 {
+    font-size: 4rem;
+}
+
+h2 {
+    font-size: 3rem;
+}
+
+h3 {
+    font-size: 2.5rem;
+}
+
+h4 {
+    font-size: 2rem;
+}
+
+li {
+    font-size: 2rem;
+}
+
+h1, h2 {
+    margin: 50px 50px;
```

Comment:
this can be shortened to `margin: 50px;`

Reply:
done